mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api
+from openerp import tools
+from openerp.tools.translate import _
+
+
+class MailNotification(models.Model):
+
+    _inherit = 'mail.notification'
+
+    @api.model
+    def get_signature_footer(
+            self, user_id, res_model=None, res_id=None, user_signature=True):
+        """
+        Override this method to add name of notified partners into the mail
+        footer
+        """
+        footer = super(MailNotification, self).get_signature_footer(
+            user_id, res_model=res_model, res_id=res_id,
+            user_signature=user_signature)
+        partner_ids = self.env.context.get('partners_to_notify')
+        if footer and partner_ids:
+            partners = self.env['res.partner'].browse(partner_ids)
+            partners_name = [
+                partner.name for partner in partners if
+                partner.notify_email != 'none'
```

Comment:
does it really make sense? If not notify_email, the partner should not even be in the mail in the first place.
I think I miss a checkbox set up in the partner "Do not communicate mail in notifications".